lib/Db/Item.php

```diff
@@ -63,6 +63,11 @@ class Item extends Entity implements IAPI, \JsonSerializable
     protected $unread = false;
     /** @var bool */
     protected $starred = false;
+    /** @var string */
+    protected $sharedBy = '';
+    /** @var string */
+    protected $sharedWith = '';
```

Comment:
This should be null if not set. Same issue as with the routes too, is it shared with Facebook or with a person? We don't know based on this name.

Reply:
This would also improve the SQL queries. `shared_by = \'\'` could become `shared_by IS NULL` which is much cleaner.